submitter/tx_submitter.go

```diff
@@ -34,119 +36,108 @@ func NewTxSubmitter(cfg *config.Config, executor *executor.Executor, daoManager
 }
 
 func (s *TxSubmitter) SubmitTransactionLoop() {
+	s.cachedEventHash = make(map[uint64][]byte, math.MaxInt)
+	submitAttempt := 0
```

Comment:
The name `submitAttempt` is a bit misleading here. It is used to clear/purge the cache after some loop times. It is not about retry or submit attempt.

Reply:
fixed